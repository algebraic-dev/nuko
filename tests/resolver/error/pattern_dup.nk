type Kek =
    | Lel Int Kek
    | Lil Int

let a =
    match a with
        Kek.Lel a (Kek.Lil a) => 2
        b => 3