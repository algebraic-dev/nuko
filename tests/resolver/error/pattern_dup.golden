This CannotFindModule "Kek"
    ( Range
        { start = Pos
            { line = 6
            , column = 8
            }
        , end = Pos
            { line = 6
            , column = 11
            }
        }
    )
